core/drivers/crypto/caam/utils/utils_sgt.c

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * Brief   Scatter-Gatter Table management utilities.
+ */
+#include <caam_common.h>
+#include <caam_utils_sgt.h>
+#include <tee/cache.h>
+
+/*
+ * Cache operation on SGT table
+ *
+ * @op     Cache operation
+ * @insgt  SGT table
+ */
+void caam_cache_op_sgt(enum utee_cache_operation op, struct caamsgtbuf *insgt)
+{
+	uint8_t idx;
```

Comment:
How about using `unsigned int` instead of a fixed width type?
Please initialize

Reply:
ok